draft-ietf-quic-tls.md

```diff
@@ -326,17 +326,12 @@ encryption levels:
 
 - CRYPTO frames MAY appear in packets of any encryption level except 0-RTT.
 
-- CONNECTION_CLOSE MAY appear in packets of any encryption level other than
```

Comment:
Section 10.3 of the transport draft says that if 1-RTT keys are not available, CONNECTION_CLOSE can be sent in Initial and Handshake, but if I read this right, this change means that CONNECTION_CLOSE can only be sent in 1-RTT and 0-RTT?

Reply:
Given we can't stateless reset the connection until the handshake is complete, I really want to continue allowing CONNECTION_CLOSE in Initial and Handshake.

Reply:
+1, I feel like we've had this discussion several times and all agreed that CONNECTION_CLOSE needs to be allowed in Initial and Handshake packets.

Reply:
Arghh, I need to revert this one.